### PR TITLE
Fixed a few typos

### PR DESCRIPTION
**Pull request by pouyakary**
While reading the code fixed a few typos

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -4,7 +4,8 @@ environment:
   sdk: '>=2.14.0 <3.0.0'
 
 dev_dependencies:
-  collection: ^1.16.0
   path: ^1.8.2
   string_scanner: ^1.1.1
   test: ^1.21.4
+dependencies:
```
Comment (devoncarew) [inline pubspec.yaml:10]: Why this change?

File: test/golden_test.dart [hunk 2]
```diff
@@ -116,5 +116,5 @@ String _buildGoldenText(String content, List<ScopeSpan> spans) {
   return buffer.toString();
 }
 
-/// Normalises newlines in code for comparing.
+/// Normalizes newlines in code for comparing.
```
Comment (devoncarew) [inline test/golden_test.dart:119]: This is technically just a UK / US locale thing (fine to change though).

**Review by devoncarew — COMMENTED** [review]
Thanks for the contribution! One question about the pubspec change.

File: test/support/span_parser.dart [hunk 3]
```diff
@@ -106,11 +106,11 @@ class ScopeSpan {
 
   bool contains(int token) => (start <= token) && (token < end);
 
-  /// Splits the current [ScopeSpan] into multiple spans separated by [cond].
+  /// Splits the current [ScopeSpan] into multiple spans separated by [condition].
```
Comment (DanTup) [inline test/support/span_parser.dart:109]: The source for this file is in DevTools and will be synced to her periodically if fixes are made, so these changes would need to be contributed there to avoid being overwritten.

https://github.com/dart-lang/dart-syntax-highlight/blob/0df49894ac85b1b44d9120ebbcb01ab418974fea/test/support/span_parser.dart#L5-L8

Comment (pouyakary) [reply test/support/span_parser.dart:109]: Oh I see, okay...
